scaffolding-ruby/README.md

```diff
@@ -0,0 +1,5 @@
+# Habitat Ruby Scaffolding
+
+* [Habiat Website](https://www.habitat.sh/)
```

Comment:
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO! It was the FIRST WORD!